src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheck.java

```diff
@@ -95,7 +95,6 @@ public void setMax(int max) {
     @Override
     public void beginTree(DetailAST rootAST) {
         context = new Context(null);
-        contextStack.clear();
```

Comment:
We push and pop context from context stack, so context stack may remain not empty only if visitTocken is called and leaveToken is not called, could you help with a test case where this condition is fulfilled?